package.json

```
@@ -20,6 +20,7 @@
     "react-scripts": "5.0.1",
     "react-spring": "9.7.1",
     "sass": "1.55.0",
```

```suggestion
    "sass": "1.59.3",
```
Esta es la que usamos el resto

---

Eso es pq estas en main. E snormal, cuando lo mergee se actualizara